.github/workflows/ci.yml

```diff
@@ -2,6 +2,20 @@ name: CI
 on:
   pull_request: {}
 jobs:
+  spotless:
+    name: spotless
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: eskatos/gradle-command-action@v1
+        name: gradle
+        with:
+          arguments: spotlessCheck -PspotlessFrom=origin/${{ github.base_ref }}
+          wrapper-cache-enabled: true
+          dependencies-cache-enabled: true
+          configuration-cache-enabled: true
```

Comment:
I was wondering why don't we run `spotless` with the other checks?

Reply:
yeah indeed we could do that. one specificity is that the other jobs only need to checkout the commit (depth 1) but spotless needs to have access to the base branch so it needs a full checkout (depth 0). nothing prevents us from doing so in one of the others 3 existing jobs though.

it can go both ways. I thought that having it as a separate job would make it cleared what failed. this could be turned into a `license check` _step_ inside the `other tests` _job_ instead of a separate job.

Reply:
I was thinking about this limits https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration

Reply:
job concurrency limit? got it. updated the PR with your suggestion. The `reactor-core` one should probably do the same.